snuba/settings/settings_test.py

```diff
@@ -19,3 +19,7 @@
 # something and you're at your wit's end, try setting this to False
 # to explore the unrefined Expression structure
 PRETTY_FORMAT_EXPRESSIONS = True
+
+# Admin
+SLACK_API_TOKEN = os.getenv("SLACK_API_TOKEN")
+SNUBA_SLACK_CHANNEL_ID = os.getenv("SNUBA_SLACK_CHANNEL_ID")
```

Comment:
I don't think these lines are needed. I think this is only if we are running with different values in tests. As far as I can tell these aren't being used in tests and are not different from the original ones.

Reply:
ahhhh okays yeah i'll take it out